hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeysDeleteRequest.java

```diff
@@ -121,13 +124,15 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager, TermIn
             .setVolumeName(volumeName).setBucketName(bucketName);
 
     boolean deleteStatus = true;
+    long startNanos = Time.monotonicNowNanos();
```

Comment:
Since there are multiple variables defined: `startNanos`, `startNs`, `startNano`, it should be a good idea to rename these variables based off of the latency they are capturing, maybe something like: `startNanosDeleteKeySuccessLatencyNs`.